install/0000_80_machine-config-operator_01_machineconfig.crd.yaml

```diff
@@ -335,6 +335,10 @@ spec:
                 description: Contains which kernel we want to be running like default
                   (traditional), realtime
                 type: string
+              overrideImage:
```

Comment:
we may need a bit better name later on because using this method we could be overriding as well as layering rpm.